TriangleTask/EquaterialTriangleBuilder.java

```diff
@@ -0,0 +1,18 @@
+/**
+ * Builds equateterial triangle.
+ */
+public class EquaterialTriangleBuilder {
+  /**
+   * Builds equaterial triangle.
+   * @return exemplar of successor of Triangle.
+   */
+  public Triangle triangleBuild(String[] sequence) {
+    try {
+      Triangle triangle = new EquaterialTriangle(sequence);
+      return triangle;
+    } catch (IllegalArgumentException a) {
+      IsoscelesTriangleBuilder triangleBuilder = new IsoscelesTriangleBuilder();    
```

Comment:
If it's attempt to create chain of responsibility - then it's implemented in wrong way, every object in chain must contain link to other object and must not know particular class of next object